src/Security/Core/OAuthUserProvider.php

```diff
@@ -2,17 +2,72 @@
 
 namespace App\Security\Core;
 
+use App\Entity\User;
 use HWI\Bundle\OAuthBundle\OAuth\Response\UserResponseInterface;
-use HWI\Bundle\OAuthBundle\Security\Core\User\FOSUBUserProvider;
+use HWI\Bundle\OAuthBundle\Security\Core\User\EntityUserProvider;
 use HWI\Bundle\OAuthBundle\Security\Core\Exception\AccountNotLinkedException;
+use Symfony\Component\Security\Core\Exception\UsernameNotFoundException;
 
-class OAuthUserProvider extends FOSUBUserProvider
+class OAuthUserProvider extends EntityUserProvider
 {
     /**
      * {@inheritdoc}
      */
     public function loadUserByOAuthUserResponse(UserResponseInterface $response)
     {
+        dump($this->em->getRepository(User::class));
```

Comment:
@angelk Can I get some help here? Don't know why, I cannot retrieve the User:class repository.

Reply:
works on my machine :)
Whats the error? Try clearing the cache

```
OAuthUserProvider.php on line 18:
App\Repository\UserRepository {#500 ▼
  #_entityName: "App\Entity\User"
  #_em: Doctrine\ORM\EntityManager {#385 …11}
  #_class: Doctrine\ORM\Mapping\ClassMetadata {#453 ▶}
}
```

Reply:
Clearing the cache did the job.